module-2/backend/week_two.md

```diff
@@ -14,8 +17,13 @@ end
 
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
 
+*We can call all, find, new, create, count, find_by, and where because the class is set up to inherit methods from ActiveRecord.*
+
 3. Assume that in your database, a team has the following attributes: "id", "name", owner_id". How would you find the name of a team with an id of 4? Assuming your class only included the code from question 2, how could you find the owner of the same team?
 
+*Team.find(4).name*
+*Team.find(4).owner after adding belongs_to: owner to the Team model.*
```

Comment:
This is right (consistent with your answer below), and definitely how we want to do it!

Also want you to be able to handle some unconventional situations, so (even though we would *never* do this), you could potentially do the following:

```ruby
owner_4_id = Team.find(4).owner_id
Owner.find(owner_4_id)
```